fvm/meter/weighted/meter.go

```diff
@@ -199,9 +199,7 @@ func WithComputationWeights(weights ExecutionEffortWeights) WeightedMeterOptions
 func WithMemoryWeights(weights ExecutionMemoryWeights) WeightedMeterOptions {
 	if int(common.MemoryKindLast)-len(weights) != 1 {
 		return func(m *Meter) {
-			for kind, weight := range weights {
```

Comment:
@dsainati1 This was causing the tests to fail, as the the memory weights are assigned like this:
```
		memoryWeights:          DefaultMemoryWeights,
```

so this code caused changes to the `DefaultMemoryWeights`